airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -65,7 +65,8 @@ def use_cache(self) -> bool:
         return False
 
     def request_cache(self) -> requests.Session:
-        cache_dir = Path(os.getenv(ENV_REQUEST_CACHE_PATH))
+        # Defaults to current directory to ensure this doesn't fail when running from unit tests or environment that do not use the entrypoint
+        cache_dir = Path(os.getenv(ENV_REQUEST_CACHE_PATH, "."))
```

Comment:
this isn't enough. we need to delete the files if ENV_REQUEST_CACHE_PATH does not exist